doc/development/deprecations.rst

```diff
@@ -39,16 +39,50 @@ Pending deprecations
   - Deprecated in v0.43
   - Will be removed in v0.44
 
-* The boolean functions provided by ``pennylane.operation`` are deprecated. See below for alternate code to
-  use instead.
+* Accessing ``lie_closure``, ``structure_constants`` and ``center`` via ``qml.pauli`` is deprecated. Top level import and usage is advised.
```

Comment:
Took me a while to figure out what happened here with the diffs lol

Reply:
Yeah, unfortunately I couldn't find any solution for now: it's quite a fundamental comprimise happening in `git diff` algorithm so...